libs/explore/src/lib/explore-map-view.component.ts

```diff
@@ -139,6 +154,9 @@ export class ExploreMapViewComponent extends AsyncHandler implements OnInit {
         return this._settings.get('app.explore.legend') || EMPTY;
     }
 
+    public readonly map_key_exists: Observable<boolean> =
```

Comment:
Rename this to be easier to understand, `map_key_exists` does tell my what it's for at a glance.
Something like `use_mapindoors` or `mapindoors_api_available`.

Reply:
I've now updated this to `use_mapsindoors`